onnx/defs/training/defs.cc

```diff
@@ -138,9 +138,9 @@ auto-differentiation.
 
 ONNX_PREVIEW_TRAINING_OPERATOR_SET_SCHEMA(
     Gradient,
-    1,
+    22,
```

Comment:
Not 22, may be 2? This is a different domain.

Reply:
good catch. fixed